### PR TITLE
[ISSUES-2677]Add ResourceServiceImplTest unit test

### PR DESCRIPTION
**Pull request by binbinCheng**
Add ResourceServiceImplTest unit test
Related issue #2677

### CODE REVIEW

File: linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+    @Mock private VersionDao versionDao;
+
+    @BeforeEach
+    @DisplayName("Each unit test method is executed once before execution")
+    public void before() throws Exception {}
+
+    @AfterEach
+    @DisplayName("Each unit test method is executed once before execution")
+    public void after() throws Exception {}
```
Comment (casionone) [inline linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java:56]: pelase remove unuse code

File: linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+    Resource addResource() {
```
Comment (casionone) [inline linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java:58]: add名字不准确 
建议使用 buildResource

File: linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+    }
+
+    @Test
+    public void testCheckResourceId() throws Exception {
+        // TODO: Test goes here...
+        String resourceId = "123";
+        Mockito.when(resourceDao.checkExists(resourceId)).thenReturn(1);
+        boolean boo = resourceServiceImpl.checkResourceId(resourceId);
+        assertTrue(boo == true);
```
Comment (casionone) [inline linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java:123]: assertTrue(boo) is ok?

File: linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java (hunk 2 above)
Comment (binbinCheng) [reply linkis-public-enhancements/linkis-bml/linkis-bml-server/src/test/java/org/apache/linkis/bml/service/ResourceServiceImplTest.java:58]: ok

**Pull request merged**